versions/l-/libgnutls.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "089f1c103a3f2c52e6ae54e8956a98345502e286",
+      "git-tree": "09f2d8c5e4e07d2076324767d251fef3bc4acb8c",
+      "version": "3.6.15",
+      "port-version": 1
+    },
+    {
+      "git-tree": "738722ca178b60ce49b2e7581ace1adf054c1144",
```

```suggestion
      "git-tree": "089f1c103a3f2c52e6ae54e8956a98345502e286",
```
Previously published versions should not be modified

---

Fixed in  #16807